test/test_dependencies.jl

```diff
@@ -0,0 +1,47 @@
+using Mimi
+
+#list of URLs of branches of packages to test
+dependencies = ["https://github.com/davidanthoff/fund.jl/archive/master.zip"]
+
+#list of failed tests to build as you go
+errors = []
+
+#make a temporary directory to run the tests in
+run(`mkdir tmp_testing`)
+cd("tmp_testing")
+
+#loop through each dependent package
+for url in dependencies
+  #download the package
+  run(`wget $url`)
+  zip_name = chomp(readall(`basename $url`))
+  run(`unzip -x $zip_name`)
```

Comment:
Two options here: there might be a package that does this in a cross-platform way. Otherwise you have to use ``7z`` on windows and ``unzip`` on unix/mac.